### DISCUSSION TITLE
Is Avalonia ready for .net 6 ?

### DISCUSSION THREAD

**Discussion by flarive34rs** [AvaloniaUI/Avalonia#6859] [Q&A] [answered]
.net 6 final is going to be released in a few days with a lot of performance improvements.
Is Avalonia 0.10.10 compatible and have been tested with .net 6 ?

Thanks for the good work, i really love Avalonia !

**Comment by ahopper** [comment, marked as answer]
I've been releasing my app for the last few months using .net 6  on win and mac with no issues (approx 1000 users) . I've been using net5.0 on linux purely because the packaging tool I use does not yet support net6.0.  but it runs locally with 6.

**Comment by maxkatz6** [comment]
For future, every new .NET (Core) version is compatible with the previous one. It's very rare situation when there is compatibility issue. 
So, at this point I can say, that Avalonia is ready not only for .NET 6, but also 7 and probably even 8, since we are not using deprecated features.

**Reply by maxkatz6** [reply]
Which doesn't mean, that Avalonia is ready for some *new* features available in new runtimes.

**Reply by flarive34rs** [reply]
Ok, great !
Thanks
